scripts/generator/graphql_generator/csharp/SDK/iOS/iOSNativeCheckout.ApplePayEventReceiver.cs.erb

```diff
@@ -44,7 +48,22 @@ namespace <%= namespace %>.SDK.iOS {
         }
 
         public void DidFinishCheckoutSession(string serializedMessage) {
-            //TODO
+            var message = NativeMessage.CreateFromJSON(serializedMessage);
+            var paymentStatus = (NativePaymentStatus) Enum.Parse(typeof(NativePaymentStatus), (string) message.Content);
```

Comment:
So our assumption here is that `message.Content` will only contain `"Success", "Cancelled", "Failed"` correct?